COMET.Web.Common.Tests/Components/BookEditor/EditotPopupTestFixture.cs

```diff
@@ -47,6 +47,8 @@ namespace COMET.Web.Common.Tests.Components.BookEditor
 
     using NUnit.Framework;
 
+    using RichardSzalay.MockHttp;

```

Comment:
Where is this dependency comming from?

Reply:
This is used for MockHttpMessageHandler (also on other fixtures in the project). Either way, one of the tests I modified had this using but it wasn't in use so I've removed it.